pkg/controlplane/instance.go

```diff
@@ -677,6 +678,7 @@ var (
 	betaAPIGroupVersionsDisabledByDefault = []schema.GroupVersion{
 		autoscalingapiv2beta1.SchemeGroupVersion,
 		autoscalingapiv2beta2.SchemeGroupVersion,
+		apidiscoveryv1beta1.SchemeGroupVersion,
```

Comment:
this doesn't belong in this list, right? it's not a normal API group and versioned endpoint, so it's not relevant to `--runtime-config`